docs/recipes/README.md

```diff
@@ -20,7 +20,7 @@ tremor-based solutions.
 
 Make sure your environment is setup for building tremor:
 
-[Tremor Quick Start](../development/quick-start.md)
+[Tremor Quick Start](/community/development/quick-start)
```

Comment:
It could be confused for a general quick start, but this one is scoped to setting up your dev environment for tremor

```suggestion
[Tremor Development Quick Start](/community/development/quick-start)
```

Reply:
merged!